packages/@okta/vuepress-site/docs/guides/build-custom-ui-mobile/before-you-begin/android/sample.md

```diff
@@ -0,0 +1 @@
+If you want to skip this guide and just download a working sample app, download our [Android example](https://github.com/okta/samples-android/tree/master/browser-sign-in).
```

Comment:
This needs to be: https://github.com/okta/samples-android/tree/master/custom-sign-in